### PR TITLE
help regarding users of wsl2  ubuntu terminal in windows

### PR DESCRIPTION
**Pull request by r0ushann**
i did faced a hectic time running my minikube start command from linux-ubuntu wsl2 teminal in windows ,so , i stepped forward with this issue as i was not able to find a specific supporting guide for wsl2. I know the approach of mine was quite naive ,but i'm sure here are many who can sort this issue!

### CODE REVIEW

File: site/content/en/docs/drivers/_index.md [hunk 1]
```diff
@@ -38,3 +38,4 @@ To do so, we use the [Docker Machine](https://github.com/docker/machine) library
 * [VirtualBox]({{<ref "virtualbox.md">}}) - VM
 * [VMware Workstation]({{<ref "vmware.md">}}) - VM
 * [SSH]({{<ref "ssh.md">}}) - remote ssh
+* [wsl2] - you may install chocolatey in your windows powershell and run chocolatey command and then install kubectl.exe and run the " minikube start " command in your ubuntu wsl2 terminal.
```
Comment (medyagh) [inline site/content/en/docs/drivers/_index.md:41]: this page is for drivers, I would suggest adding it to a page about WSL

plus "minikube kubectl --" could be an alternative to not installing kubectl

Comment (r0ushann) [reply site/content/en/docs/drivers/_index.md:41]: A seperate page for wsl will be great!

**Comment by r0ushann** [comment]
"I signed it" - the CLA

**Comment by klaases** [comment]
Hi @r0ushann, based on yours and @medyagh's comment(s), did you want to create a separate page for WSL?

**Comment by r0ushann** [comment]
Yes, i did wanted to have a separate page for wsl highlighting its key requirements and configuration commands.

**Comment by spowelljr** [comment]
Hi @r0ushann, because this PR has been open for a while with no changes I'm going to close it. But if you find time to continue working on it feel free to reopen it, thanks!

**Pull request closed without merging**